x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -241,45 +231,27 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		details["badge"] = fmt.Sprint(badgeSig)
 		details["clientFee"] = "0"
 		details["reliabilityPay"] = "false"
-		details["Mint"] = details["BasePay"]
-
-		// Mint to module
-		if !rewardCoins.AmountOf(epochstoragetypes.TokenDenom).IsZero() {
-			err = k.Keeper.bankKeeper.MintCoins(ctx, types.ModuleName, rewardCoins)
-			if err != nil {
-				// panic:ok: mint coins should never fail
-				utils.LavaFormatPanic("critical: failed to mint coins to reward provider", err,
-					utils.Attribute{Key: "provider", Value: providerAddr},
-					utils.Attribute{Key: "reward", Value: rewardCoins},
-				)
-			}
-
-			err = k.distributeRewards(ctx, providerAddr, relay.SpecId, uint64(relay.Epoch), reward.TruncateInt())
-			if err != nil {
-				return nil, utils.LavaFormatError("could not distribute rewards for provider and delegators", err)
-			}
-		}
-
+		details["Mint"] = "0ulava"
 		details["relayNumber"] = strconv.FormatUint(relay.RelayNum, 10)
+		details["rewardedCU"] = strconv.FormatUint(rewardedCU, 10)
+		// differentiate between different relays by providing the index in the keys
+		successDetails := appendRelayPaymentDetailsToEvent(details, uint64(relayIdx))
+		// calling the same event repeatedly within a transaction just appends the new keys to the event
+		utils.LogLavaEvent(ctx, logger, types.RelayPaymentEventName, successDetails, "New Proof Of Work Was Accepted")
 
-		cuAfterQos := uint64(reward.Quo(coinsPerCu).TruncateInt64())
+		cuAfterQos := uint64(rewardedCUDec.TruncateInt64())
 		err = k.chargeComputeUnitsToProjectAndSubscription(ctx, clientAddr, relay, cuAfterQos)
```

Comment:
should change the name since it's not only charging CUs from the project but also awaring it to the provider

Reply:
done. see https://github.com/lavanet/lava/commit/3b37e3ad41d3e548c74b458e0a85170abd0134b0